### ISSUE TITLE
training issue:loss=nan when use 2B model

### ISSUE THREAD

**Issue by ucaswindlike** [aigc-apps/VideoX-Fun#82]
when using 2b model,loss=nan after first step. using 5B model did not meet this issue.

**Comment by bubbliiiing** [comment]
We haven't encountered this situation yet. Are you using BF16?

**Comment by ucaswindlike** [comment]
yes,encounter this when using bf16, while fp32 is running correctly
